sensor_msgs/Image snapshot # Snapshot to process
---
sensor_msgs/Image match_contour # Best matching contour in the request Image
float64 match_cost # cost of matching match_contour to the DB
